admin-dev/themes/new-theme/js/pages/product/combination/filters/Filters.vue

```diff
@@ -57,81 +57,109 @@
 </template>
 
 <script lang="ts">
-  import FilterDropdown from '@pages/product/combination/filters/FilterDropdown.vue';
-  import ProductEventMap from '@pages/product/product-event-map';
   import {defineComponent, PropType} from 'vue';
+  import CheckboxesDropdown from '@app/components/checkboxes-dropdown/CheckboxesDropdown.vue';
+  import EventEmitter from '@components/event-emitter';
+  import ProductEventMap from '@pages/product/product-event-map';
+  import ProductMap from '@pages/product/product-map';
+  import {Attribute, AttributeGroup} from '@pages/product/combination/types';
+  import {Choice} from '@app/components/checkboxes-dropdown/types';
 
   const CombinationEvents = ProductEventMap.combinations;
 
+  interface Filter {
```

Comment:
I wonder if this interface should be in `@app/components/checkboxes-dropdown/types` as well? Since it's kind of used to "drive" the dropdown component it maybe useful for other future usages as well?

Reply:
hmm no, I see it as another implementation of dropdown component. So now we have the reusable dropdown component, which is implemented in 2 places - 1. carriers checkboxes dropdown and 2. combination filters dropdonws.